Dockerfile-manylinux2014_x86_64

```diff
@@ -9,7 +9,9 @@ ENV PATH /opt/python/cp27-cp27mu/bin/:${PATH}
 
 ENV PREFIX /usr/local
 ENV PKG_CONFIG_PATH=${PREFIX}/lib/pkgconfig
-ENV ESSENTIA_3RDPARTY_VERSION v2.1_beta5-46-g896a402f
+ENV ESSENTIA_3RDPARTY_VERSION 8ae940e717c6b6565a3e988ba834aca38f0c4246
```

Comment:
Use a more reader-friendly version number using git describe (just like the one used previously)